### ISSUE TITLE
idek what this is

### ISSUE THREAD

**Issue by KenWa51** [PrismarineJS/mineflayer#777]
```
Chunk size is 167 but only 41 was read ; partial packet : {"name":"named_sound_effect","params":{"soundName":"minecraft:register","soundCategory":118,"x":1767994981,"y":1920166255,"z":1849322597,"volume":1.064367405277846e+24,"pitch":2.1234712599405015e-10}}; buffer :19126d696e6563726166743a726567697374657276696176657273696f6e3a6c65676163792f697a6775792d2d2d0076696176657273696f6e3a6c65676163792f697a6775793763696b6d2d2d2d2d2d2d0076696176657273696f6e3a6c65676163792f697a6876657232660062756e676565636f72643a6d61696e0076696176657273696f6e3a6c65676163792f6a6a68756b713273696664666937636b6a356575342d2d2d
Chunk size is 21 but only 2 was read ; partial packet : {"name":"entity","params":{"entityId":0}}; buffer :25000003f9000000000003640764656661756c7400
Chunk size is 73 but only 41 was read ; partial packet : {"name":"named_sound_effect","params":{"soundName":"minecraft:register","soundCategory":118,"x":1767994981,"y":1920166255,"z":1849322597,"volume":1.064367405277846e+24,"pitch":2.123044240409655e-10}}; buffer :19126d696e6563726166743a726567697374657276696176657273696f6e3a6c65676163792f696e62712d2d2d2d0076696176657273696f6e3a6c65676163792f6b35637567766b6a
Chunk size is 48 but only 38 was read ; partial packet : {"name":"named_sound_effect","params":{"soundName":"minecraft:brand","soundCategory":30,"x":1114992231,"y":1701135215,"z":1919164456,"volume":3.0223280890057526e+32,"pitch":7.657597288305564e+28}}; buffer :190f6d696e6563726166743a6272616e641e42756e676565436f72642028756e6b6e6f776e29203c2d20537069676f74
```

Attempted to join a server using the bot.

then the bot simply quits after the console is presented with that information ^
[nodemon] clean exit - waiting for changes before restart

**Comment by KenWa51** [comment]
I resolved this issue by adding   version:"1.8", to the bot creation

**Comment by KenWa51** [comment]
New issue though

bot.on("spawn", function(){

  console.log("spawned in")

})

For some reason spawn doesnt fire?
